src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java

```diff
@@ -17,25 +21,33 @@
 import uk.gov.companieshouse.web.accounts.controller.ConditionalController;
 import uk.gov.companieshouse.web.accounts.exception.ServiceException;
 import uk.gov.companieshouse.web.accounts.model.directorsreport.AddOrRemoveDirectors;
+import uk.gov.companieshouse.web.accounts.model.directorsreport.DirectorToAdd;
 import uk.gov.companieshouse.web.accounts.model.state.CompanyAccountsDataState;
 import uk.gov.companieshouse.web.accounts.service.smallfull.DirectorService;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+import java.util.List;
 
 @Controller
 @NextController(ProfitAndLossQuestionController.class)
 @PreviousController(DirectorsReportQuestionController.class)
 @RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/add-or-remove-directors")
 public class AddOrRemoveDirectorsController extends BaseController implements ConditionalController {
 
+
     @Autowired
     private HttpServletRequest request;
 
     @Autowired
     private DirectorService directorService;
 
+    AddOrRemoveDirectors addOrRemoveDirectors = new AddOrRemoveDirectors();
```

Comment:
Should this be private AddOrRemoveDirectors addOrRemoveDirectors with @Autowired annotation ?